.ci/robot_framework/tests/tests_005_basics.robot

```diff
@@ -13,3 +13,7 @@ Check Kernel Configuration available in /proc/config.gz
     Log    ${stdout}
     Should Contain    ${stdout}[0]    Kernel Configuration
 
+Check L2 Cache is enabled
+    ${TEST_BOARD_IP}    Get Environment Variable    TEST_BOARD_IP
+    ${stdout}=    SSH Command    ${TEST_BOARD_IP}   /root/scripts/check-l2-cache-is-enabled.sh
+    Should Be Equal As Strings  ${stdout}  L2 cache enabled
```

Comment:
According with the report, the output is a tuple: 

`${stdout} = ('L2 cache enabled', '')`

```suggestion
    Should Be Equal As Strings  ${stdout}[0]  L2 cache enabled
```

Reply:
Maybe there is some room for improvement in https://github.com/Igalia/meta-wpe-image/blob/main/.ci/robot_framework/libs/TestUtils.py#L29 since the `SSH command` is our own implementation for a SSH command execution.

Reply:
commented on this, I think the rest is as complete for the current casuistic.